doc/microtex-doc.toml.in

```diff
@@ -0,0 +1,47 @@
+[library]
+version = "@version@"
+description = "A dynamic, cross-platform, and embeddable LaTeX rendering library"
+authors = "NanoMichael <artiano@hotmail.com>"
```

Comment:
I suggest adding yourself too since you are one of the main maintainers of this lib. And I'll make a contributor list for this project.

Reply:
I'm seemingly unable to insert any linebreak there :/ using a comma is as good as it gets.